pyaedt/modeler/Primitives3D.py

```diff
@@ -1050,8 +1050,10 @@ def create_udm(self, udmfullname, udm_params_list, udm_library="syslib"):
         vArg1.append("ConnectionID:=")
         vArg1.append("")
         oname = self.oeditor.CreateUserDefinedModel(vArg1)
+        obj_list = self.oeditor.GetPartsForUserDefinedModel(oname)
+        self.modeler.user_defined_models[oname] = obj_list
```

Comment:
@Samuelopez-ansys here you are setting a property (user_defined_models) but that property doesn't have a setter.